api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -83,13 +83,39 @@ public CohortDefinition getE() {
     return cd;
   }
 
+  /**
+   * SqlCohortDefinition for Viral Load Result
+   *
+   * @return
+   */
+  public CohortDefinition getViralLoadResult() {
```

Comment:
@simaomazive please use a more suggestive name like 
```suggestion
  public CohortDefinition getNumberOfPatientsInFichaClinicaWithViralLoadResults() {
```

Reply:
@simaomazive like here https://github.com/jembi/openmrs-module-eptsreports/blob/develop/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java#L101